docs/vault/guide.md

```diff
@@ -137,6 +137,17 @@ Now you must send the BTC to the Bitcoin address **generated by the parachain**.
 
 You Vault only starts earning rewards once BTC is locked - and rewards are determined by your share of the total BTC locked in the system. Hence, you can increase your rewards by bringing your own BTC into the system.
 
+### How to avoid sniping while minting?
+- You can go to polkadot.js.org/apps and deactivate accepting new issue requests from Developer -> Extrinsics -> vaultRegistry -> acceptNewIssues(KSM, KBTC, No)
+- After 36 hours (normally 24 hours, but blocktimes), all the pending issue request without a BTC transaction will be cancelled
+- No one can mint against your collatreal while issuing is turned false so you can add collateral without a worry.
+- For self-minting, you can batch:
+1. Developer -> Extrinsics -> vaultRegistry -> acceptNewIssues(KSM, KBTC, true)
+2. Developer -> Extrinsics -> issue -> requestIssue
+3. Developer -> Extrinsics -> vaultRegistry -> acceptNewIssues(KSM, KBTC, false)
+![Screenshot: Creating batchAll call to avoid sniping your collateral](../_assets/img/vault/batchallcall.png)
```

Comment:
```suggestion
### How to avoid sniping while self-minting?
*Sniping* is the name given by the community to the minting of kBTC / iBTC immediately following new locked collateral by a Vault. This prevents Vaults from maintaining a collateralization threshold that is higher than the system-defined [secure threshold](#secure-collateral). Vault operators have control over whether to accept new issue requests or not, so sniping is prevented when issue requests are disabled.

!> In the following example, the Vault is assumed to use `KSM` as collateral. If your Vault is using a different collateral currency, replace the value accordingly.

Go to polkadot.js.org/apps and deactivate new issue requests by submitting the following extrinsic: `Developer` -> `Extrinsics` -> `vaultRegistry` -> `acceptNewIssues(KSM, KBTC, No)`.


For self-minting, batch the following extrinsics:
1. `vaultRegistry` -> `acceptNewIssues(KSM, KBTC, true)`
2. `issue` -> `requestIssue`
3. `vaultRegistry` -> `acceptNewIssues(KSM, KBTC, false)`

![Screenshot: Creating batchAll call to avoid sniping your collateral](../_assets/img/vault/batchallcall.png)
```